frontend/src/components/PdfContainer.tsx

```diff
@@ -0,0 +1,28 @@
+import { pdf } from "src/constants/nofoPdfs";
+
+import Image from "next/image";
+import Link from "next/link";
+import { Grid } from "@trussworks/react-uswds";
+
+const PdfContainer = ({ file, image, alt }: pdf) => {
+  return (
+    <Grid className="margin-bottom-2" tablet={{ col: 6 }} tabletLg={{ col: 3 }}>
```

Comment:
nit: `PdfContainer` seems like the wrong name. It doesn't contain any PDFs. It links to PDFs, but technically that doesn't even have to be the case; you could pass it whatever link (`file` doesn't even have to be a file). Functionally this component if more of an "image link." You might even move the grid element into the `nofoPdfs.map` in the funding content and let this component just worry about making image links.